### DISCUSSION TITLE
Tag of meilisearch

### DISCUSSION THREAD

**Discussion by anoordover** [CommunityToolkit/Aspire#291] [General]
According to dockerhub the tags latest, v1, v1.11 and v1.11.3 for meilisearch have the same hashcode. Should we use v1 or do we risc getting breaking changes when we are on the major version?
Bigger question: When should we use which tag?

**Comment by aaronpowell** [comment]
I think we want to stick to minor tags, so 1.11 in this case, so that we are working on a "known stable" line of a container image. Using a patch version, such as 1.11.3, would result in too frequent a churn on the image and just using the major version would risk too broad a window for changes to be incorporated.

Since Aspire exposes the API to override the defaults, there's always more flexibility for consumers to adjust to a tag that better fits their update cadence.
